### DISCUSSION TITLE
Two questions not answered in the FAQ:

### DISCUSSION THREAD

**Discussion by metaleap** [Poly-Haven/polyhaven.com#74] [Q&A] [answered]
### Q1: "Missing geometry (tree models)"

FAQ [states](https://docs.polyhaven.com/en/faq#missing-geometry-tree-models) this:

> Some of our assets make use of Blenders Geometry Nodes. More specifically the leaves on the trees are "instances/duplicates" of smaller leaf models and currently exporting the model without making the instanced geometry real results in the missing geometry not found in the FBX, GLTF, and USD formats.

> We are working on fixing this issue, but

I'd like to Follow that Issue, is there no Github Issue number for this? I skimmed over both open and closed Issues' titles just now and nothing seems to fit.

### Q2: "Resolution" for models

As a bit of a newbie, I was wondering what exactly the 1K, 2K, 4K, 8K designations might mean for _model-type assets_. I get it for _texture-type assets_, but for models? Does that also refer to a model's own included/embedded textures? I suppose so, but would like to be sure.  =)  Also, what's common for use with game dev / a game engine's real-time rendering. Certainly 4K and 8K sound overkill but for a whole lot of 3D art 2K also might be overkill and yet for much other 3D art it might be not but indeed the quality minimum. Some models are big, some small, some only ever in the distance, some get to be seen extremely close-up etc..... does everyone keep all 1K/2K/4K versions around to more freely decide on the fly during design iterations?

**Comment by gregzaal** [comment, marked as answer]
Hi there,

# Q1

There is no GitHub issue for this, but we plan to dedicate some time to it later this year when we have some other projects out of the way.

# Q2

Yes, resolutions for models are to do with their texture resolutions. The mesh data is all the same, but the image textures are different resolutions.

I would recommend using the 1K assets, and then if you find the resolution to be too pixelated for your use case, download a higher resolution.
